src/district/server/smart_contracts.cljs

```diff
@@ -1,6 +1,5 @@
 (ns district.server.smart-contracts
   (:require
-    [cljs-web3.core :as web3]
```

Comment:
web3 is actually user. Take a look at this line https://github.com/rmuslimov/district-server-smart-contracts/blob/f5a9a9ba47b7862c7ffa91a58c6ffb339903d643/src/district/server/smart_contracts.cljs#L146

Reply:
Yes, but it's redefined few lines below: https://github.com/district0x/district-server-smart-contracts/blob/master/src/district/server/smart_contracts.cljs#L11

Reply:
It's not redefined. Clojure is smart enough to differentiate namespace and symbol even if they have same name. If you use `web3/` it always knows you're referring to the namespace.